test/subclass-test.js

```diff
@@ -1,20 +1,22 @@
+'use strict';
```

Comment:
Is this line needed?

Reply:
Not anymore now that babel is around. Removed.